src/pages/Profile.jsx

```diff
@@ -108,7 +108,7 @@ const Profile = () => {
           variant="contained"
           color="secondary"
           size="small"
-          aria-label="Add Client Button"
+          aria-label="Add Contact Button"
```

Comment:
Not adding a client/contact here - I commented on this in Brandon's PR... This is adding a document button.

Reply:
Ah, good catch. I see it now. I'll have this fixed.